.github/workflows/nodejs.yml

```diff
@@ -14,12 +14,12 @@ jobs:
         node-version: [10.x, 14.x, 16.x, 18.x]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
let's do this in a separate PR, unclear what the differences are yet and this is an orthogonal change